docs/css/main.css

```diff
@@ -0,0 +1,308 @@
+body {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  width: 100%;
+}
+
+code {
+  background-color: var(--color-extra-light-gray);
+  color: var(--secondary-color);
+  font-weight: var(--weight-bold);
+  padding-inline: var(--size-2);
+}
+
+p {
+  color: var(--primary-color-dark);
+  font-size: calc(1.2 * var(--size-9));
+  line-height: 1.3;
+  padding-left: var(--size-3);
+}
+
+/* Scroll effects offset */
+
+[id] {
+  scroll-margin-top: 7em;
+}
+
+.main {
+  color: var(--primary-color);
+  font-size: var(--size-11);
+  margin: auto;
+  max-width: calc(75 * var(--size-9));
+}
+
+main {
+  gap: var(--size-11);
+  grid-template-columns: 1fr 1fr;
+  min-height: 90vh;
+}
+
+iframe {
+  border: var(--size-3) double var(--color-dark-gray);
+}
+
+/* Hero image */
+
+.hero-img {
+  margin-inline: auto;
+  width: 55%;
+}
+
+.hero-text {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+/* Hero text */
+
+.hero-text a button {
+  color: var(--primary-color);
+}
+
+.hero-text a button:hover {

```

Comment:
it's better to give button a specific class instead of targeting it directly. It might cause problem if you decided to use this styles in you ECOM project.

Reply:
Thanks for the suggestion, but these styles are only for the landing page. But I'll change it just in case.